src/db/airtable.ts

```diff
@@ -1,4 +1,8 @@
 import Airtable from 'airtable';
+import dotenv from 'dotenv';
+
+dotenv.config({ path: '.env.local' });
+dotenv.config();
```

Comment:
We need to use dotenv? 
(Because it's a good tool but nextJS can already read the .env file.)

Reply:
to be able to run scripts